gateway-ha/src/main/java/io/trino/gateway/ha/config/MonitorConfiguration.java

```diff
@@ -133,4 +136,14 @@ public void setMetricMaximumValues(Map<String, Float> metricMaximumValues)
     {
         this.metricMaximumValues = metricMaximumValues;
     }
+
+    public int getMetricRefreshSeconds()
+    {
+        return metricRefreshSeconds;
+    }
+
+    public void setMetricRefreshSeconds(int metricRefreshSeconds)
+    {
+        this.metricRefreshSeconds = metricRefreshSeconds;
+    }
```

Comment:
You can use `Duration` here to allow for any units instead of fixing to seconds, see this example:
https://github.com/trinodb/trino/blob/ac18d5bc061f6cf4283660f39dba60534af85bba/core/trino-main/src/main/java/io/trino/execution/scheduler/NodeSchedulerConfig.java#L190-L201